### PR TITLE
Admin Error View

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: packages/admin/src/components/ErrorView/ErrorView.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    max-width: 650px;
+    margin: 0 auto;
+  }
+`
+
+const ErrorView = ({message, technicalReason}) => {
+  return <StyledErrorView>
+    <Icon icon="times" style={{color: '#C01E30', fontSize: '80px'}}/>
+    <Typography.H1><FormattedMessage id={message || 'client.admin.errorView.defaultTitle'}/></Typography.H1>
+    <Typography.Span>{technicalReason}</Typography.Span>
```
Comment (rzueger) [inline packages/admin/src/components/ErrorView/ErrorView.js:21]: maybe don't render span if technicalReason isn't set or doesn't it matter? or is this prop required?

File: packages/admin/src/routes/entities/modules/path/sagas.js [hunk 2]
```diff
@@ -120,7 +133,14 @@ export function* loadCurrentViewInfo({payload: {location}}) {
...
 
-        yield call(addRecordToBreadcrumbs, path.entity + '/' + path.key, path.entity, path.key, breadcrumbs)
+        const bc = yield call(
+          addRecordToBreadcrumbs, path.entity + '/' + path.key, path.entity, path.key, breadcrumbs
+        )
+
+        if (bc && bc.error) {
+          yield put(actions.setCurrentViewInfo(pathname, {...currentViewInfo, error: bc.error}))
+          return
+        }
```
Comment (rzueger) [inline packages/admin/src/routes/entities/modules/path/sagas.js:143]: this method starts looking dangerous... or continues.. 🙃 💀

Comment (MethodenMann) [reply packages/admin/src/routes/entities/modules/path/sagas.js:143]: yes thats why i created a task to refactor and add unit tests. see TOCDEV-2326

**Review by PadPalon — COMMENTED** [review]
Seems good to me, rzs comment seem relevant

**Pull request merged**